content/api/rest-api.textile

```diff
@@ -506,11 +506,18 @@ bc[json]. {
       transportType: <string>,
       <additional recipient address key/value pairs>
     },
-    state: <active or failed>
+    state: <active or failed>,
+    error: {
+      code: <string>,
+      statusCode: <string>,
+      message: <string>
+    }
   }
 }
 
-or a 404 error if a device by that ID does not exist.
+or a 404 error if a device by that ID does not exist. 
+
+If Ably could not register the device on a behalf of the push notifications provider, then @push.error@ contains the relevant error information.
```

Comment:
"If Ably could not register the device on **~~a~~**  behalf of the push notifications provider, then @push.error@ contains the relevant error information."

Extraneous 'a'.

Reply:
well spotted! :)